elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -335,6 +336,22 @@ protected RequestScope(String path, JsonApiDocument jsonApiDocument, RequestScop
     }
 
     /**
+     * Merge the global filterExpression with the one provided.
+     * @param filterExpression the filterExpression to merge
+     * @return The global filter expression evaluated at the first load
+     */
+    public Optional<FilterExpression> addGlobalFilterExpression(FilterExpression filterExpression) {
```

Comment:
Methods that begin with the word 'add' imply that they mutate the object.  This method doesn't do that.   In retrospect, this probably doesn't belong in RequestScope.